corehq/apps/reports/views.py

```diff
@@ -957,6 +957,19 @@ def rebuild_case_view(request, domain, case_id):
 @require_case_view_permission
 @require_permission(Permissions.edit_data)
 @require_POST
+def resave_case(request, domain, case_id):
+    case = get_document_or_404(CommCareCase, domain, case_id)
+    CommCareCase.get_db().save_doc(case._doc)  # don't just call save to avoid signals
```

Comment:
@czue Are we only doing this for when something goes wrong in elasticsearch (i.e., all the post-save signals already ran but something got messed up with the case pillow so it's not in sync)? If so, then it's fine to not reprocess the reminder post-save signals (it doesn't cause any harm to do so, but it will create unnecessary overhead, especially if we do this in bulk for a bunch of cases).